main/settings/src/main/scala/sbt/Scope.scala

```diff
@@ -99,18 +99,69 @@ object Scope {
       case BuildRef(uri) => BuildRef(resolveBuild(current, uri))
     }
 
-  def display(config: ConfigKey): String = config.name + ":"
+  def display(config: ConfigKey): String = guessConfigIdent(config.name) + "/"
+
+  private[sbt] val configIdents: Map[String, String] =
+    Map(
+      "it" -> "IntegrationTest",
+      "scala-tool" -> "ScalaTool",
+      "plugin" -> "CompilerPlugin"
+    )
+  private[sbt] val configIdentsInverse: Map[String, String] =
+    configIdents map { _.swap }
```

Comment:
It's a matter of taste. I prefer using `{}` for an infix expression that accepts a lambda, as it feels like it's part of a language construct. Probably a discussion we should have in another issue.